src/xml_document.cc

```diff
@@ -317,7 +317,7 @@ xmlParserOption getParserOptions(v8::Local<v8::Object> props) {
     ret |= getParserOption(props, "cdata", XML_PARSE_NOCDATA, false);       // 16384: merge CDATA as text nodes
 
     ret |= getParserOption(props, "noxincnode", XML_PARSE_NOXINCNODE);      // 32768: do not generate XINCLUDE START/END nodes
-    ret |= getParserOption(props, "xinclude", XML_PARSE_NOXINCNODE, false); // 32768: do not generate XINCLUDE START/END nodes
+    ret |= getParserOption(props, "xincnode", XML_PARSE_NOXINCNODE, false); // 32768: do not generate XINCLUDE START/END nodes
 
```

Comment:
As the commit message describes, I am having trouble getting xinclude to work, and tried to fix it with this change. This *looks* like a typo to me: the `xinclude` property is already defined above, and I think this one should be `xincnode`. But, it didn't fix my problem, and so I'm not sure. If I'm wrong, let me know, and I'll take this out of the PR.